test/Sdk.Analyzers.Tests/Sdk.Analyzers.Tests.csproj

```diff
@@ -7,19 +7,26 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.6.3" />
-    <PackageReference Include="SmartAnalyzers.RoslynTestKit" Version="4.7.0" />
-    <PackageReference Include="xunit" Version="2.4.2" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.5">
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.8.0" />
+    <PackageReference Include="SmartAnalyzers.RoslynTestKit" Version="5.0.2" />
+    <PackageReference Include="System.Data.SqlClient" Version="4.8.6" />
```

Comment:
Did you mean to add `System.Data.SqlClient`? Where do we use it?

Reply:
Same question for `System.Drawing.Common`

Reply:
Yes. Those have been promoted from transitive dependencies.